engine/src/main/java/com/sandboni/core/engine/ResultGenerator.java

```diff
@@ -119,6 +121,30 @@ private Object getContent(ResultContent rc) {
                 return graphOperations.getChangeStats();
             case FORMATTED_CHANGE_STATS:
                 return graphOperations.getFormattedChangeStats();
+            case RELATED_UNIT:
+                return graphOperations.getUnitRelatedTests();
+            case RELATED_CUCUMBER:
+                return graphOperations.getCucumberRelatedTests();
+            case DISCONNECTED_UNIT:
+                return graphOperations.getUnitDisconnectedTests();
+            case DISCONNECTED_CUCUMBER:
+                return graphOperations.getCucumberDisconnectedTests();
+            case RELATED_EXTERNAL_UNIT:
+                return graphOperations.getUnitRelatedExternalTests();
+            case RELATED_EXTERNAL_CUCUMBER:
+                return graphOperations.getCucumberRelatedExternalTests();
+            case ALL_EXTERNAL_UNIT:
+                return graphOperations.getAllExternalUnitTests();
+            case ALL_EXTERNAL_CUCUMBER:
+                return graphOperations.getAllExternalCucumberTests();
+            case ALL_TESTS_SIZE:
```

Comment:
The 'size' entries (RELATED_TESTS_SIZE, ALL_TESTS_SIZE, DISCONNECTED_TESTS_SIZE)should not be here - whoever wants the size can take the returned list's size

Reply:
I will remove RELATED_TESTS_SIZE and DISCONNECTED_TESTS_SIZE. But ALL_TESTS_SIZE should be here - maven plugin doesn't know about total test count in the project.